src/Database/Behaviors/Purgeable.php

```diff
@@ -76,12 +76,7 @@ public function purgeAttributes($attributesToPurge = null)
         $cleanAttributes = array_diff_key($attributes, array_flip($purgeable));
         $originalAttributes = array_diff_key($attributes, $cleanAttributes);
 
-        if (is_array($this->originalPurgeableValues)) {
-            $this->originalPurgeableValues = array_merge($this->originalPurgeableValues, $originalAttributes);
-        }
-        else {
-            $this->originalPurgeableValues = $originalAttributes;
-        }
+        $this->originalPurgeableValues = array_merge($this->originalPurgeableValues, $originalAttributes);
```

Comment:
Why was this change made? Does this property always exist as at least an array? If so where is it defined?

Reply:
It will always be an array, so the condition that was used previously is redundant.

Lines 68-70 above will convert a single string provided to the argument into an array, and the `purgeable` property itself is defined as an array.